pkg/maps/ipcache/ipcache.go

```diff
@@ -248,3 +250,21 @@ var (
 func Reopen() error {
 	return IPCache.Map.Reopen()
 }
+
+// Function to update IPCache map with node PodCIDR, VTEP CIDR
+func UpdateIPCacheMapping(newCIDR *cidr.CIDR, newTunnelEndpoint net.IP,
```

Comment:
nit: If this is specific to VTEP, maybe it's worth calling that out in the function name? Otherwise the next developer to come along may get confused between the purpose of this function vs. `Upsert()`.

```suggestion
// Function to update IPCache map with node PodCIDR, VTEP CIDR
func UpdateIPCacheVTEPMapping(newCIDR *cidr.CIDR, newTunnelEndpoint net.IP,
```

Reply:
will do